FRENCH/src/foreword.md

```diff
@@ -12,10 +12,10 @@ domains than you did before.
 -->
 
 Cela n'a pas toujours été aussi évident, mais le langage de programmation Rust
-apporte avant tout plus de *puissance* : peu importe quel type de code vous êtes
-en train d'écrire actuellement, Rust vous permet d'aller plus loin et de
-programmer en toute confiance dans une plus grande diversité de domaines que
-vous n'auriez pu le faire auparavent.
+apporte avant tout plus de *puissance* : peu importe le type de code que vous
+écrivez en ce moment, Rust vous permet d'aller plus loin et de
```

Comment:
`à un moment donné`, instead ?

Reply:
I'm not sure.
The original text uses `now` and the *be + -ing* form to say that it is currently the case, whereas `à un moment donné` would imply that it could be the case at any moment, not necessarily right now.